tests/Http/Api/ActivityApiTest.php

```diff
@@ -291,10 +291,14 @@ public function testQuantityOnActivityIndex()
         $thirdSignup->quantity_pending = null;
         $thirdSignup->save();
 
-        for ($i = 0; $i < 3; $i++) {
-            $post = $thirdSignup->posts()->save(factory(Post::class)->make());
-            $post->quantity = 3;
+        $posts = factory(Post::class, 3)->create();
+        $postsTotalQuantity = [];
+
+        foreach ($posts as $post) {
+            $post->signup()->associate($thirdSignup);
             $post->save();
+
+            array_push($postsTotalQuantity, $post->quantity);
```

Comment:
And then, _I think_, to sum up all the quantities, you can grab the posts as a collection and use `sum` 

`$signup->posts->sum('quantity');`

Reply:
omg yes this is WAY better - thanks @sbsmith86 ! Updated :)